src/console.h

```diff
@@ -14,6 +14,9 @@ void console_init();
 void console_putchar( char c );
 int  console_write( int unit, const void *buffer, int nblocks, int offset );
 void console_heartbeat();
+void console_set_fgcolor(uint8_t r, uint8_t g, uint8_t b);
+void console_set_bgcolor(uint8_t r, uint8_t g, uint8_t b);
+uint8_t console_verify_color_range(uint8_t x);
```

Comment:
Nope. I asked last Friday about whether to expose "private" functions in the headers and was told to leave the declaration in the .h anyways.